pkg/retry/retry.go

```diff
@@ -124,12 +124,20 @@ func (t ticker) Stop() {
 // ExpectedError error represents an error that is expected by the retrying
 // function. This error is ignored.
 func ExpectedError(err error) error {
+	if err == nil {
```

Comment:
added this as shortcut, so that we could do `return ExpectedError(someFunc())` without caring about return status in `RetryableFunc`